src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -57,12 +57,21 @@ const Body = BaseForm.extend(Object.assign(
         : loc('mfa.calling', 'login');
       const enterCodeText = loc('oie.phone.verify.enterCodeText', 'login');
       const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
+      const isPhoneNumberAvailable = this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login');
+      let nicknameText = isPhoneNumberAvailable ? this.model.get('nickname') : '';
+      let extraNicknameCssClasses = '';
+      if (nicknameText !== '') {
+        nicknameText = ' (' + nicknameText + ')';
+        extraNicknameCssClasses = 'strong no-translate authenticator-enrollment-nickname';
+      }
 
       const strongClass = this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
         'strong no-translate' : '';
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">
-        ${sendText}&nbsp;<span class='${strongClass}'>${this.model.escape('phoneNumber')}.</span>
+        ${sendText}&nbsp;<span class='${strongClass}'>${this.model.escape('phoneNumber')}</span>
+        <span ${ extraNicknameCssClasses ? 'class="' + extraNicknameCssClasses + '"' : ''}>
+        ${nicknameText}.</span>
```

Comment:
You should probably only add the `nickName` span element if the nickname value exists, (it also depends on if the phone number is provided). I dont think it will cause a problem visually to have an empty span here but that may be an a11y issue. 

Can you create a variable above `this.add(...` logic, and set the span with the nickName only if nickName is set. If it is not set, you can leave it as an empty string.

Then, in this area, you can just append the variable, so regardless on if it exists or not, it will render properly.

Reply:
Addressed.